docs/source/explore/get-started/vr-controls.rst

```diff
@@ -15,15 +15,114 @@ Use these controls on your VR controllers so that you can enjoy your experience
+| *Movement Direction:*      | When selected, your avatar will move in the direction your head is facing.      |
+| HMD-Relative               |                                                                                 |
++----------------------------+---------------------------------------------------------------------------------+
+| *Movement Direction:*      | When selected, your avatar will move in the direction your hand is pointing.    |
+| Hand-Relative              |                                                                                 |
++----------------------------+---------------------------------------------------------------------------------+
+| *Movement Direction:*      | When selected, your avatar will move in the direction your hand is pointing,    |
+| Hand-Relative (Level)      | without taking pitch into account.                                              |
++----------------------------+---------------------------------------------------------------------------------+
+| Dominant Hand              | Select 'Left' or 'Right'. Teleport and turning controls move to the controller  |
+|                            | in the dominant hand.                                                           |
++----------------------------+---------------------------------------------------------------------------------+
+| *Control Scheme Selection:*| In Default mode, your walking speed doesn't change depending on how far forward |
+| Default                    | you push your controller's joystick. Fully pushing your joystick forward will   |
+|                            | make your avatar run at twice your walking speed.                               |
++----------------------------+---------------------------------------------------------------------------------+
+| *Control Scheme Selection:*| In Analog mode, your walking speed changes based on how far forward you push    |
+| Analog                     | your controller's joystick. Fully pushing your joystick forward will            |
+|                            | make your avatar run at twice your walking speed.                               |
++----------------------------+---------------------------------------------------------------------------------+
+| *Control Scheme Selection:*| In Analog++ mode, your walking speed changes based on how far forward you push  |
+| Analog++                   | your controller's joystick. You can also use the slider to change the walking   |
+|                            | speed in meters/second. Fully pushing your joystick forward will make your      |
+|                            | avatar run at twice your walking speed.                                         |
++----------------------------+---------------------------------------------------------------------------------+
 
 -------------------------
 Gamepad
```

Comment:
Does Gamepad belong in the Desktop page? I feel like its not likely to be used with an HMD, but rather with a computer...

Reply:
This seems to something no one is really agreeing on - gamepads are also used for VR experiences. I'm going to leave it as it is and create a manuscript case for this.